mobile/native/src/orderbook.rs

```diff
@@ -109,6 +110,15 @@ pub fn subscribe(
                                 tracing::info!("Received a rollover request from orderbook.");
                                 event::publish(&EventInternal::BackgroundNotification(BackgroundTask::Rollover(TaskStatus::Pending)));
 
+                                // Sending a failed event with a delay. This event will be ignored by the UI if the task has
+                                // been successful in the meantime.
+                                tokio::spawn(async {
+                                    tokio::time::sleep(ROLLOVER_PROTOCOL_TIMEOUT).await;
+                                    event::publish(&EventInternal::BackgroundNotification(
+                                        BackgroundTask::RecoverDlc(TaskStatus::Failed),
```

Comment:
```suggestion
BackgroundTask::Rollover(TaskStatus::Falied)
```

perhaps this is what  you wanted?
I honestly can't tell because the commit message is unrelated 😞

Reply:
oh yeah.. good catch 👍  but as discussed above, I think that code should be anyways moved to the frontend.

Reply:
also:  sorry for the typo with `Falied` in my suggestion 😭